File: Scala/alg03.scala
/* ALG03 */
import scala.io.StdIn
import scala.math.pow
object Estudo extends App {
  var X: Int = 0
  var Y: Int = 0
  X = scala.io.StdIn.readInt()
  Y = pow(X, 2).toInt
  println(Y)
}

File: C#/alg03.cs
using System;
namespace ALG03
{
    class Program
    {
        static void Main(string[] args)
        {
            int X;
            int Y;
            X = int.Parse(Console.ReadLine());
            Y = (int)Math.Pow(X, 2);
            Console.WriteLine(Y);
        }
    }
}
